.github/workflows/ci.yaml

```diff
@@ -25,13 +25,22 @@ jobs:
       CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
     steps:
     - uses: actions/checkout@v2
+    - name: Set TRAVIS ENV variables
+      run: |
+        echo "TRAVIS_BRANCH=${{ github.base_ref || github.ref_name }}" >> $GITHUB_ENV
+    - name: Install system packages
+      run: sudo apt-get install -y libcurl4-openssl-dev
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby-version }}
-        bundler-cache: true
     - name: Prepare tests
       run: bin/setup
+    - name: Set up bundler caching
```

Comment:
you are doing this twice?

Reply:
I'm experimenting in this PR...ignore the man behind the curtain 😆